kernel/initrd.c

```diff
@@ -9,6 +9,10 @@ void *initrd_end;
 
 extern void process_run(const char *image_start, unsigned long image_size);
 
+
+/// \file       kernel:initrd.c
+/// \brief      initrd_load loads initrd information and initialzes the size
```

Comment:
Please move these file comments at the top of the file (before anything else) so when someone reads the file in a text editor, they will see it first.

Reply:
I think these tags should be just above the function, otherwise after generating docs we won't see description with a method name. If we move the line above, we can use `\fn` tag. Please let me know.